domains/transfer/Transfer.API/API.IntegrationTests/Transfer/Some.cs

```diff
@@ -1,4 +1,4 @@
-namespace API.UnitTests;
+namespace API.IntegrationTests.Transfer;
 
 public static class Some
```

Comment:
The intention of this is to have easy access to test objects which is not a Transfer specific thing. Put it outside of transfer